test/ts/index.test.tsx

```diff
@@ -0,0 +1,19 @@
+import { h } from 'preact'
```

Comment:
Is there another way to check types match with `tsc`? I want to try and avoid writing type system tests.

Reply:
nope, I mean, how do you want test that something works without writing assertion for that :)
 
what I can do, is to move tests within `/types` folder where all TS stuff will reside, similar for `flow`

having no test for correct type declaration is a big no no ( YMMV ofc ), because without them, we are giving consumers absolutely no guarantee that our definitions are correct and that they work.